libs/template/spec/render-strategies/render-strategies/native.strategy.spec.ts

```diff
@@ -0,0 +1,82 @@
+// tslint:disable-next-line:nx-enforce-module-boundaries
+import { mockConsole } from '@test-helpers';
+import { getStrategies } from '@rx-angular/template';
+import { getMockNativeStrategyConfig } from '../../fixtures';
+import { from, of } from 'rxjs';
+import createSpy = jasmine.createSpy;
+
+const nativeStrategyName = 'native';
+
+describe('native Strategy', () => {
+  beforeAll(() => mockConsole());
+
+  it('should be present in strategies map', () => {
+    const strategy = getStrategies(getMockNativeStrategyConfig())[nativeStrategyName];
+    expect(strategy).toBeDefined();
+  });
+
+  it(`should have ${nativeStrategyName} as name`, () => {
+    const strategy = getStrategies(getMockNativeStrategyConfig())[nativeStrategyName];
+    expect(strategy.name).toBe(nativeStrategyName );
+  });
+
+  it('should call cdRef#markForCheck 1 time when rxScheduleCD is used with a single sync emission', (done) => {
```

Comment:
Should we separate checks for `markForCheck` & `detectChanges` into different tests? (applicable for all tests)

Reply:
Good idea!